test/e2e/specs/specs-playwright/wp-invite__new-user.ts

```diff
@@ -0,0 +1,131 @@
+import {
+	DataHelper,
+	EmailClient,
+	LoginFlow,
+	SidebarComponent,
+	InvitePeoplePage,
+	PeoplePage,
+	setupHooks,
+	UserSignupPage,
+	Roles,
+	BrowserManager,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe.skip( DataHelper.createSuiteTitle( `Invite: New User` ), function () {
```

Comment:
As noted in the description, the test is skipped by default, just as it was for Selenium.

This test is too expensive resource-wise to run every commit and would be better served running only once per trunk merge.